frontend/src/features/admin-form/settings/components/PaymentSettingsSection/BusinessInfoSection.tsx

```diff
@@ -88,26 +88,18 @@ const BusinessInfoBlock = ({
   }
   return (
     <>
+      {settings.payments_field.gst_enabled ? (
+        <FormControl mb="2.5rem" isReadOnly={mutateFormBusiness.isLoading}>
+          <FormLabel isRequired>GST Registration Number</FormLabel>
+          <BusinessFieldInput
+            placeholder={agencyDefaults?.gstRegNo || ''}
+            initialValue={settings.business?.gstRegNo || ''}
+            handleMutation={handleGstRegNoMutation}
+          />
+        </FormControl>
+      ) : null}
       <FormControl mb="2.5rem" isReadOnly={mutateFormBusiness.isLoading}>
-        <FormLabel
-          description="Leave blank to use your agency's GST Registration Number"
-          isRequired
-        >
-          GST Registration Number
-        </FormLabel>
-        <BusinessFieldInput
-          placeholder={agencyDefaults?.gstRegNo || ''}
-          initialValue={settings.business?.gstRegNo || ''}
-          handleMutation={handleGstRegNoMutation}
-        />
-      </FormControl>
-      <FormControl mb="2.5rem" isReadOnly={mutateFormBusiness.isLoading}>
-        <FormLabel
-          description="Leave blank to use your agency's business address"
```

Comment:
I see that the description is removed as well! Safe to assume that this was done with input from design?

Reply:
Yep! Checked with @staceytan1998